src/main/scala/org/graphframes/GraphFrame.scala

```diff
@@ -32,76 +32,188 @@ import org.graphframes.pattern._
 
 
 /**
+ * A representation of a graph using [[DataFrame]]s.
+ *
  * Represents a graph with vertices and edges stored as [[DataFrame]]s.
```

Comment:
Oh, I really meant the 2nd paragraph "Represents..." but we can update it later.